### DISCUSSION TITLE
new open source app that is similar

### DISCUSSION THREAD

**Discussion by zombiehoffa** [babybuddy/babybuddy#717] [General]
someone just announced something similar on lemmy: https://lemmy.world/post/6439632

More informational than anything.

**Comment by cdubz** [comment]
Nice! Thanks for sharing. The space is generally much more crowded then back when I first started Baby Buddy but this is the most mature self-hostabled, privacy focused solution I've seen in the same space. It looks more focused on the privacy side of the things than the self-hosting but that's not necessarily a bad thing 😄 UI looks good!
